ml_model/src/main/java/edu/wm/sealab/featureextraction/FeatureVisitor.java

```diff
@@ -61,6 +63,19 @@ public void visit(ConstructorDeclaration cd, Void arg) {
   public void visit(IfStmt ifStmt, Void arg) {
     super.visit(ifStmt, arg);
     features.incrementNumOfIfStatements();
+    features.setConditionals(features.getConditionals() + 1);
+  }
+
+  /**
+   * This method to compute # switch statements of a java method (not entries)
+   *
+   * @param SwitchStmt
+   * @param Void
+   */
+  @Override
+  public void visit(SwitchStmt swStmt, Void arg) {
+    super.visit(swStmt, arg);
+    features.setConditionals(features.getConditionals() + 1);
```

Comment:
use an increment method instead of a setter

Reply:
Do you want me to do this for all of the other features previously implemented by others?